### ISSUE TITLE
Sidebar property selector: keyboard and tooltip accessibility

### ISSUE THREAD

**Issue by aviannamiller** [omeka/omeka-s#2421]
The property selector sidebar is not keyboard accessible. The text input can be entered and interacted with via keyboard (to filter down properties), but this does not help much in terms of making a selection as content within the selector component itself cannot be accessed or activated via keyboard.

In addition to this, tooltips provided next to property names are not accessible. In addition to the lack of keyboard accessibility in this pattern, the tooltips are not hoverable or dismissible (unclear if they have the capacity to be persistent since I can't test via keyboard). [#2405 (Provide accessible tooltips instead of relying on title attribute)]( https://github.com/omeka/omeka-s/issues/2405) discusses solutions for the inaccessible tooltip issue as a whole and would likely be resolved first.
